services/roomService/src/lib/Maze.ts

```diff
@@ -53,4 +44,11 @@ export default class Maze {
   removeGame(gameId: string): void {
     this._gameIds.delete(gameId);
   }
+
+  /**
+   * Returns true if the maze contains the specified game
+   */
+  hasGame(gameId: string): boolean {
+    return this._gameIds.has(gameId);
+  }
```

Comment:
but also if we're going to write something like this I don't see why gameIds should be public, when we could also just add a getGameIds to return the private list?

Reply:
Oh, these might've been added when Tanvi was trying to write tests.  I'll fix it.  Nice catch